src/hooks/useUSDPrice.ts

```diff
@@ -41,8 +41,8 @@ function useETHValue(currencyAmount?: CurrencyAmount<Currency>): {
     }
   }
 
-  if (!trade || !currencyAmount?.currency || !isGqlSupportedChain(chainId)) {
-    return { data: undefined, isLoading: state === TradeState.LOADING || state === TradeState.SYNCING }
+  if (!trade || state === TradeState.LOADING || !currencyAmount?.currency || !isGqlSupportedChain(chainId)) {
```

Comment:
adding in this case because it wasn't caught before - if the state is loading, then that means the USD price could be displayed for an outdated currency and will give a flash of the wrong value